src/app/services/mail/mail.service.ts

```diff
@@ -241,10 +241,12 @@ export class MailService {
       autoReplyMailData.sender || form.admin.agency.fullName
     ).replace('(', '\\(')
 
+    const responseId = submission.id
     const defaultBody = `Dear Sir or Madam,\n\nThank you for submitting this form.\n\nRegards,\n${form.admin.agency.fullName}`
     const autoReplyBody = (autoReplyMailData.body || defaultBody).split('\n')
 
     const templateData = {
+      responseId,
```

Comment:
instead of saving `responseId` to a variable, I would much rather just do:
```
const templateData = {
  responseId: submission.id
}
```
there are a couple of reasons for this:
1. it's easier to read. currently if I'm not sure what `responseId` refers to, I have to go and find where the variable was declared. if we avoid declaring the variable, it's immediately clear that it's the submissionId.
2. we generally try to avoid holding state in variables. this is because it's hard to tell if a variable (especially an object) has been modified along the way. if we just declare them inline, it's clear that they haven't been modified.